dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -6,7 +6,16 @@ if [ "$EUID" -ne 0 ] ; then
   exit 1
 fi
 
-if [ -f /etc/sonic/chassisdb.conf ]; then
+PLATFORM="$(sonic-cfggen -d -v DEVICE_METADATA.localhost.platform)"
```

Comment:
MSFT proposed a way to check if the system is a Smart Switch by accessing the `platform.json` file. The presence of the `DPUS` table means that the system is a Smart Switch NPU. The presence of the `DPU` table means that the system is a Smart Switch DPU: [Smart Switch IP Address Assignment](https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md#device-data-and-platform).

This approach is used in various places by the Smart Switch Database service. I suggest using it here as well for consistency and to minimize the number of files that describe the system.

Reply:
E.g. https://github.com/sonic-net/sonic-buildimage/pull/17562

Reply:
Will follow that

Reply:
Created https://github.com/sonic-net/sonic-buildimage/pull/19729  and addressed these comments  in #19729